webrtc.html

```diff
@@ -783,6 +783,14 @@
                     then reject <var>p</var> with a
                     <code>InvalidStateError</code> and abort these steps.</p>
                   </li>
+
+                  <li> <p>If the content of <var>description</var> is not valid
+                  SDP syntax, then reject <var>p</var> with an
+                  <code>RTCError</code> with errorDetail set to "sdpSyntaxError"
+                  and the sdpLineNumber attribute set to the line number in the
+                  SDP where the syntax error was detected then abort these
+                  steps.</p> </li>
```

Comment:
Lets say some SDP had 4 m line sections, the first is audio, second is main video, and next two came from polycom 3 screen system with proprietary extensions that are not valid SDP. If we an error on a given line, we will often just remove the whole m-section that line was in and try again. We might have look at the line and remove it and try again.